src/cli/Microsoft.DotNet.xunit.performance.analysis.cli/project.json

```diff
@@ -33,6 +33,8 @@
     }
   },
   "runtimes": {
-    "win7-x64": {}
+    "win7-x86": {},
```

Comment:
I still have reservations about explicitly adding "win7-x86" as a runtime. Do we have x86 machines for CI? I think we should ask someone from the dotnet cli team to verify this for us. 
@brianrob thoughts?